interface/themes/style.php

```diff
@@ -70,15 +70,37 @@ function validate_html_color_code($arg) {
 
 }
 
-td, tr, .table, .bgcolor1,  ul.tabNav, .navbar, .nav, .dropdown, .navbar-header, input[type='submit'], ul.tabNav a, .navbar-collapse{
+.table, .bgcolor1,  ul.tabNav, .navbar, .nav, .dropdown, .navbar-header, ul.tabNav a, .navbar-collapse{
 
   background-color: #$secondary_color;
   color: #$secondary_font_color;
 }
 
-input[type='submit'], input[type='button'],input[type='submit'], button, a[role='button'] {
-	background-color: #$button_color;
-	color: #$button_font_color;
-	border: 2px solid #$button_font_color;
-}";
+.cp-positive {
```

Comment:
@aethelwulffe  these are temporary until the ui for changing them gets done.

Reply:
OK.  Got it.